apps/css-workshop/package.json

```diff
@@ -4,6 +4,13 @@
   "private": true,
   "version": "0.1.0",
   "description": "Workshop environment containing HTML pages demonstrating itwinui-css",
+  "dependencies": {
```

Comment:
nit: everything should be under devDependencies, as per the established convention